scripts/__tests__/stryker-security-config.test.mjs

```
@@ -63,4 +65,22 @@ describe("security mutation Stryker configuration", () => {
+
+  // Every assertion above is about the DECLARED scope, and every one of them was true while the
+  // scheduled lane on `dev` still aborted in its dry run on that exact functional pipeline (#3349).
+  // Stryker's vitest runner does not filter with `testFiles`; it hands execution to vitest, which
+  // collects from the vitest config's own include set. So the declaration has to be checked against
+  // the config the lane actually runs, or this file keeps proving an intent nobody enforces.
+  it("runs a vitest config that actually excludes the functional pipeline", async () => {
+    const config = await loadSecurityMutationConfig();
+    const configFile = config.vitest?.configFile;
+
+    expect(configFile).toBe("vitest.mutation.config.ts");
+    const source = await readFile(new URL(`../../${configFile}`, import.meta.url), "utf8");
```

ESLint

[error] 'URL' is not defined.

(no-undef)